use-cases/vehicle-routing-capacity/src/main/java/org/acme/vehiclerouting/rest/VehicleRoutingSolverException.java

```diff
@@ -0,0 +1,30 @@
+package org.acme.vehiclerouting.rest;
+
+import jakarta.ws.rs.core.Response;
+
+public class VehicleRoutingSolverException extends RuntimeException {
```

Comment:
I am ok with adding it for now, but I believe that custom Exceptions and ExeptionMapper etc are overkill for quickstarts. Less code is more easier to grok for users that quick start.

Reply:
I see your point, but the `cUrl` experience is just so much better. If everybody thinks it's an overkill, I will remove it, though.